src/compas/topology/__init__.py

```diff
@@ -58,21 +58,48 @@
 
 """
 from __future__ import absolute_import
-from __future__ import division
-from __future__ import print_function
 
 import compas
 
-from .traversal import *  # noqa: F401 F403
-from .combinatorics import *  # noqa: F401 F403
-from .orientation import *  # noqa: F401 F403
+from .traversal import (
+    depth_first_ordering,
+    breadth_first_ordering,
+    breadth_first_traverse,
+    breadth_first_paths,
+    shortest_path,
+    astar_shortest_path,
+    dijkstra_distances,
+    dijkstra_path
+)
+from .combinatorics import vertex_coloring, connected_components
+from .orientation import face_adjacency, unify_cycles
+from .connectivity import adjacency_from_edges
 
 if compas.IPY:
-    from .orientation_rhino import *  # noqa: F401 F403
+    from .orientation_rhino import face_adjacency_rhino, unify_cycles_rhino
```

Comment:
not to change on this PR, but orientation_rhino should move to `compas_rhino` at some point

Reply:
not sure if i agree. in my opinion it is like a `numpy` backend alternative but just with `rhino`...

Reply:
But we should handle it with plugins, same as we do for booleans in which the operations are provided by compas_rhino when inside of it, otherwise they can be numpy-based

Reply:
okay so the pluggable is in the core lib in `topology` but the rhino plugin is in `compas_rhino`?

Reply:
yep!